components/profile/my-network-settings.tsx

```diff
@@ -425,10 +431,27 @@ export default function MyNetworkSettings({ network, updateEditingNetwork } : My
             value={settings?.treasury?.closeFee?.value}
             error={settings?.treasury?.closeFee?.validated === false}
             onChange={fields.closeFee.setter}
+            disabled={true}
```

Comment:
shouldn't this depend on the users wallet and if it is a network governor?

Reply:
I created a section to control this situation and left this input as default disabled for the user to see that it exists but he has no control